client/src/components/MobileAd/mobile-ad-modal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Modal } from '@freecodecamp/react-bootstrap';
+import Bowser from 'bowser';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import './mobile-ad-modal.css';
+
+function MobileAdShow(): JSX.Element {
```

Comment:
```suggestion
function MobileAd(): JSX.Element {
```
Nitpick: the 'Show' is a bit redundant.